spec/Configuration/Dotenv/ParseSpec.hs

```diff
@@ -141,5 +142,5 @@ spec = describe "parse" $ do
   it "doesn't allow more configuration options after a quoted value" $
     parseConfig `shouldFailOn` "foo='bar'baz='buz'"
 
-parseConfig :: String -> Either (ParseError Char Dec) [ParsedVariable]
+parseConfig :: String -> Either (ParseError Char Void) [ParsedVariable]
```

Comment:
Because as mentioned in he first part of the [documentation](https://hackage.haskell.org/package/megaparsec-6.0.2/docs/Text-Megaparsec.html) I defined a type alias for parser `type Parser = Parsec Void String` and `ParseError` must keep the same type. So, in this case it would be `Void`